client/src/components/SharedComponents/Signup/index.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react'
+import './signup.css'
+
+class Signup extends Component {
+  state = {
+    name: '',
+    email: '',
+    password: '',
+    confirm_password: '',
+    displayBio: false
+  }
+
+  onChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  ShowDisplayBio = () => {
+    this.setState({ displayBio: true })
+  }
+
+  lessDisplayBio = () => {
+    this.setState({ displayBio: false })
+  }
+
+  validateForm = () => {
+    return (
+      this.state.name.length > 2 &&
+      this.state.email.length > 6 &&
+      this.state.password.length > 3 &&
+      this.state.password === this.state.confirm_password
+    )
+  }
+
+  render() {
+    return (
+      <>
+        <h1>Signup</h1>
+
+        <form>
+          <input
+            type="text"
+            name="name"
+            onChange={this.onChange}
+            placeholder="name..."
+            value={this.state.name}
+            required
+          />
+          <br />
+
+          <input
+            type="email"
+            name="email"
+            onChange={this.onChange}
+            placeholder="Email..."
+            value={this.state.email}
+            required
+          />
+          <br />
+
+          <input
+            type="password"
+            name="password"
+            onChange={this.onChange}
+            placeholder="password"
+            value={this.state.password}
+            required
+          />
+          <br />
+          <input
+            type="password"
+            name="confirm_password"
+            onChange={this.onChange}
+            placeholder="confirm_password"
+            value={this.state.confirm_password}
+            required
+          />
+          <br />
```

Comment:
If this is to add white space, please use css to add margins.
```<br />``` should be used as an inline element that adds a single line space in texts.